### PR TITLE
Added required sections in sympy.combinatorics

### PR DESCRIPTION
**Pull request by Soumi7**
# Updates sympy.combinatorics to follow latest docstring style guide

#### References to other Issues or PRs

#19591

#### Brief description of what is fixed or changed
- Required sections in :
  - polyhedron.py
  - subsets.py
  - tensor_can.py
  - testutil.py
  - util.py

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/combinatorics/util.py [hunk 1]
```diff
@@ -243,22 +252,27 @@ def _orbits_transversals_from_bsgs(base, strong_gens_distr,
...
-    ``slp`` - if ``True``, return a list of dictionaries containing the
-              generator presentations of the elements of the transversals,
-              i.e. the list of indices of generators from `strong_gens_distr[i]`
-              such that their product is the relevant transversal element
+    ``base`` - The base.
+    ``strong_gens_distr`` - Strong generators distributed by membership in basic
+    stabilizers.
+    ``transversals_only`` - bool
+        A flag switching between returning only the
+        transversals/ both orbits and transversals.
```
Comment (asmeurer) [inline sympy/combinatorics/util.py:270]:
```suggestion
        transversals and both orbits and transversals.
```

File: sympy/combinatorics/util.py [hunk 2]
```diff
@@ -243,22 +252,27 @@ def _orbits_transversals_from_bsgs(base, strong_gens_distr,
...
+    ``slp`` -
+        If ``True``, return a list of dictionaries containing the
+        generator presentations of the elements of the transversals,
+        i.e. the list of indices of generators from `strong_gens_distr[i]`
```
Comment (asmeurer) [inline sympy/combinatorics/util.py:274]:
```suggestion
        i.e. the list of indices of generators from ``strong_gens_distr[i]``
```

**Comment by asmeurer** [comment]
Looks good other than those two comments.

File: sympy/combinatorics/tensor_can.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def dummy_sgs(dummies, sym, n):
 
     n : number of indices
 
-    in base form the dummy indices are always in consecutive positions
+    In base form the dummy indices are always in consecutive positions.
```
Comment (Sc0rpi0n101) [inline sympy/combinatorics/tensor_can.py:43]:
```suggestion
    In base form, the dummy indices are always in consecutive positions.
```

Comment (Sc0rpi0n101) [reply sympy/combinatorics/tensor_can.py:43]: Also, shouldn't this come under the explanation for dummies?

File: sympy/combinatorics/tensor_can.py [hunk 4]
```diff
@@ -74,8 +74,8 @@ def dummy_sgs(dummies, sym, n):
 def _min_dummies(dummies, sym, indices):
     """
     Return list of minima of the orbits of indices in group of dummies
-    see `double_coset_can_rep` for the description of `dummies` and `sym`
-    indices is the initial list of dummy indices
+    see `double_coset_can_rep` for the description of ``dummies`` and ``sym``
+    ``indices`` is the initial list of dummy indices
```
Comment (Sc0rpi0n101) [inline sympy/combinatorics/tensor_can.py:78]:
```suggestion
    see ``double_coset_can_rep`` for the description of ``dummies`` and ``sym``.
    ``indices`` is the initial list of dummy indices.
```

Comment (Sc0rpi0n101) [reply sympy/combinatorics/tensor_can.py:78]: Also, add a period at the end of the previous line.

File: sympy/combinatorics/tensor_can.py [hunk 5]
```diff
@@ -537,16 +537,19 @@ def double_coset_can_rep(dummies, sym, b_S, sgens, S_transversals, g):
...
+    Explanation
+    ===========
 
     ``base``, ``gens``  BSGS for slot permutation group
     ``g``               permutation representing the tensor
     ``num_free``        number of free indices
     The indices must be ordered with first the free indices
 
-    see explanation in double_coset_can_rep
+    See explanation in double_coset_can_rep
```
Comment (Sc0rpi0n101) [inline sympy/combinatorics/tensor_can.py:552]:
```suggestion
    See explanation in ``double_coset_can_rep``
```

File: sympy/combinatorics/testutil.py [hunk 6]
```diff
@@ -192,15 +201,20 @@ def _verify_normal_closure(group, arg, closure=None):
...
-    base_i, gens_i BSGS for tensors of this type
-    n_i  number ot tensors of type `i`
+    g : Permutation representing the tensor.
+    dummies : List of dummy indices.
+    msym : Symmetry of the metric.
+    v : A list of (base_i, gens_i, n_i, sym_i) for tensors of type `i`.
+        base_i, gens_i BSGS for tensors of this type
+        n_i  number ot tensors of type `i`
+
+    Explanation
```
Comment (Sc0rpi0n101) [inline sympy/combinatorics/testutil.py:216]: Can you move the explanation before the parameters?

File: sympy/combinatorics/polyhedron.py [hunk 7]
```diff
@@ -23,14 +26,17 @@ class Polyhedron(Basic):
     References
     ==========
 
-    http://mathworld.wolfram.com/PolyhedralGroup.html
+    .. [1] http://mathworld.wolfram.com/PolyhedralGroup.html
```
Comment (Sc0rpi0n101) [inline sympy/combinatorics/polyhedron.py:29]: Please leave an empty line after this too.

Comment (Soumi7) [reply sympy/combinatorics/polyhedron.py:29]: Done.

File: sympy/combinatorics/testutil.py (hunk 6 above)
Comment (Soumi7) [reply sympy/combinatorics/testutil.py]: Done.

File: sympy/combinatorics/tensor_can.py (hunk 5 above)
Comment (Soumi7) [reply sympy/combinatorics/tensor_can.py:552]: Changed.

File: sympy/combinatorics/tensor_can.py (hunk 4 above)
Comment (Soumi7) [reply sympy/combinatorics/tensor_can.py]: Done.

File: sympy/combinatorics/tensor_can.py (hunk 3 above)
Comment (Soumi7) [reply sympy/combinatorics/tensor_can.py]: Changed.

File: sympy/combinatorics/polyhedron.py [hunk 8]
```diff
@@ -665,7 +674,7 @@ def _pgroup_calcs():
     References
     ==========
 
-    http://dogschool.tripod.com/trianglegroup.html
+    .. [1] http://dogschool.tripod.com/trianglegroup.html
     """
```
[the remark this answers is not in the document]
Comment (Soumi7) [reply sympy/combinatorics/polyhedron.py]: Added a line in the end.

File: sympy/combinatorics/util.py (hunk 2 above)
Comment (Soumi7) [reply sympy/combinatorics/util.py]: Changed.

**Comment by Sc0rpi0n101** [comment]
Looks good to me. And looks like nobody else has any comments, so I'm merging this.

**Pull request merged**